internal/apiserver/route_post_request_message.go

```diff
@@ -41,7 +41,11 @@ var postRequestMessage = &oapispec.Route{
 	JSONOutputValue: func() interface{} { return &fftypes.MessageInOut{} },
 	JSONOutputCodes: []int{http.StatusOK}, // Sync operation
 	JSONHandler: func(r *oapispec.APIRequest) (output interface{}, err error) {
-		output, err = r.Or.SyncAsyncBridge().RequestReply(r.Ctx, r.PP["ns"], r.Input.(*fftypes.MessageInOut))
+		msg := r.Input.(*fftypes.MessageInOut)
+		if msg.Header.Group == nil && (msg.Group == nil || len(msg.Group.Members) == 0) {
```

Comment:
We've tried to avoid putting this type of functional decision logic into the router, and instead have it in the layers below. However, I understand there's no layer currently that implements this above the sync-async bridge.

My suggestion is to:
- Move these couple of lines of logic down into a common method on `orchestrator`
- Stop exposing the `SyncAsyncBridge()` function out of orchestrator, and instead make it an internal call

Reply:
Done